spec/mdn/mdn.mdown

```diff
@@ -70,17 +70,38 @@ An **MDN** object has the following properties:
 
   See [@!RFC8098] for the exact meaning of these different fields.
 
-## MDN/set
+## MDN/send
 
-This is a standard “/set” method as described in [@!RFC8620] where only create is supported; any attempt to update/destroy MUST be rejected with a `forbidden` SetError.
+The MDN/send method sends an [@!RFC5322] message from an MDN object. The capability "urn:ietf:params:jmap:mail" is implicitely required while using this method. The method takes the following arguments:
```

Comment:
I believe all people who voiced an opinion on this on the mailing list wanted that the capability has to be specified explicitly.

```suggestion
The MDN/send method sends an [@!RFC5322] message from an MDN object. When calling this method the "using" property of the Request object MUST contain the capabilities "urn:ietf:params:jmap:mdn" and "urn:ietf:params:jmap:mail". The latter because of the implicit call to Email/set, described below.
The method takes the following arguments:
```